internal/manifest/manifest.go

```diff
@@ -158,6 +164,65 @@ func Validate(manifest *Manifest) (err error) {
 	return nil
 }
 
+// LocateManifestSlices finds the paths marked with "generate:manifest" and
+// returns a map from the manifest path to all the slices that declare it.
+func LocateManifestSlices(slices []*setup.Slice, manifestFileName string) map[string][]*setup.Slice {
```

Comment:
Done in https://github.com/canonical/chisel/pull/142/commits/087d09d9a6a12ac3e4d18720a4354bcb8c1e49ce.

Reply:
The remaining part about the `manifestFileName` I wonder whether you should be able to customize it. If I want to write it to somewhere that is not `manifest.wall` then I should still be able to use this function. Maybe instead of taking manifestFileName we could return the directory and the caller is the one expected to concatenate the filename.

Reply:
No, just that I thought the helper should be more generic and that in our previous conversations we said to move the manifest filename to slicer and keep manifest generic. I will move it back here.

Reply:
Done in https://github.com/canonical/chisel/pull/142/commits/844504278c74e51d6382cdc857a014cac378e2c0.